app/views/layouts/application.html.erb

```diff
@@ -11,16 +11,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="<%=t("views.layouts.application.meta.description") %>">
     <meta property="og:type" content="website">
-    <meta property="og:url" content="<%= url_for(only_path: false) %>">
+    <meta property="og:url" content="<%= canonical_url %>">
     <meta property="og:title" content="<%= content_for(:page_title) -%> | GetYourRefund.org">
     <meta property="og:description" content="<%=t("views.layouts.application.meta.description") %>">
     <meta property="og:image" content="<%= image_url("social_share_banner.png") %>">
     <meta property="twitter:card" content="summary_large_image">
-    <meta property="twitter:url" content="<%= url_for(only_path: false) %>">
+    <meta property="twitter:url" content="<%= canonical_url %>">
     <meta property="twitter:title" content="<%= content_for(:page_title) -%> | GetYourRefund.org">
     <meta property="twitter:description" content="<%=t("views.layouts.application.meta.description") %>">
     <meta property="twitter:image" content="<%= image_url("social_share_banner.png") %>">
 
+    <link rel="canonical" href="<%= canonical_url %>" />
+    <link rel="alternate" href="<%= canonical_url(:en) %>" hreflang="x-default" />
+    <link rel="alternate" href="<%= canonical_url(:en) %>" hreflang="en" />
+    <link rel="alternate" href="<%= canonical_url(:es) %>" hreflang="es" />
```

Comment:
the canonical URL should be for the current language
the x-default and en alternates should always be english 
the es alternate should be spanish

i elected not to mess with trying to use the mireembolso domain for spanish. i think we want to focus all of our SEO on the getyourrefund domain. with this change mireembolso.org will still be indexed separately so it will show up in google search results, but we won't be telling Google to _always_ consider mireembolso.org the canonical spanish domain, so getyourrefund.org should rank higher given it has a lot more traffic. not sure if this is ideal, but it's a fine starting point.

Reply:
TIL `hreflang` - cool!